lib/KTable/index.vue

```diff
@@ -220,41 +261,81 @@
         }
         if (nextCell) {
           nextCell.focus();
```

Comment:
@BabyElias I tested and seems good to me. Would you add a comment summarizing what's the newly added code for scrolling achieving?

Reply:
Sure!

Reply:
Done